### ISSUE TITLE
Visualisation for `control_state` not showing as expected

### ISSUE THREAD

**Issue by CalMacCQ** [CQCL/tket#1102]
If I create a `QControlBox` with the `control_state=[0]` I expect to see the control as an empty circle. 

however if I run the following snippet with pytket 1.21 I get the following diagram

```python
from pytket.circuit import Circuit, CircBox, QControlBox
from pytket.circuit.display import render_circuit_jupyter

circ = Circuit(3)
A_circbox = CircBox(Circuit(2, name="A").H(0).CX(0, 1))
controlled_a_box = QControlBox(A_circbox, n_controls=1, control_state=[0])
circ = Circuit(3)
circ.add_gate(controlled_a_box, list(range(3)))

render_circuit_jupyter(circ)
```

![cA](https://github.com/CQCL/tket/assets/93673602/ab54364e-a914-43a4-b669-38557931b133)

Note the filled circle on the control qubit.

However if I instead add an extra control to the `QControlBox` I see that the diagram is now what I expect.

```python
from pytket.circuit import Circuit, CircBox, QControlBox
from pytket.circuit.display import render_circuit_jupyter

circ = Circuit(3)
A_circbox = CircBox(Circuit(2, name="A").H(0).CX(0, 1))
controlled_a_box = QControlBox(A_circbox, n_controls=2, control_state=[0, 1])
circ = Circuit(4)
circ.add_gate(controlled_a_box, list(range(4)))

render_circuit_jupyter(circ)
```

![cA2](https://github.com/CQCL/tket/assets/93673602/2acdf726-ae3a-461c-84f0-7d3a864f81e7)

Not sure what's causing this.

@DNA386

**Comment by jake-arkinstall** [comment]
I *think* I've addressed this in [this PR to pytket-circuit-renderer](https://github.com/CQCL/pytket-circuit-renderer/pull/49). control_state=0 was interpreted as a non-existent value rather than 'all controls are off', making it fall back to the default of 'all controls are on'.

**Comment by cqc-alec** [comment]
Thanks Jake! @DNA386 could you take a look at the PR please?

**Comment by CalMacCQ** [comment]
I believe this is now fixed in the [0.7.1 release](https://github.com/CQCL/pytket-circuit-renderer/releases/tag/v0.7.1) of the pytket circuit renderer.

Thanks for fixing this :)
